niche_vlaanderen/system_tables/flooding/depths.csv

```diff
@@ -1,4 +1,4 @@
-code, depth
+depth, description
```

Comment:
please remove the spaces in each of the headers and data files of the system tables csv files